src/main/scala/io/iohk/ethereum/db/dataSource/IodbDataSource.scala

```diff
@@ -1,23 +1,46 @@
 package io.iohk.ethereum.db.dataSource
 
+import java.io.File
 import java.nio.ByteBuffer
 import java.util.concurrent.atomic.AtomicLong
 
 import io.iohk.iodb.{ByteArrayWrapper, LSMStore}
 
-class IodbDataSource(lSMStore: LSMStore) extends DataSource {
+class IodbDataSource private (lSMStore: LSMStore,
+                              keySize: Int,
+                              path: String) extends DataSource {
```

Comment:
@AlanVerbner I think it would be useful to schedule such a house-keeping task, only not sure if we can afford it at the moment with so many radical changes going on.